src/color-picker/color-picker.js

```diff
@@ -26,6 +26,7 @@ const Self = class ColorPicker extends HTMLElement {
 		`;
 
 		this._el = dom.named(this);
+		this.addEventListener("propchange", this.propChangedCallback);
```

Comment:
Shouldn't we remove the corresponding listener in `disconnectedCallback()` here and in other components? Or will this listener be destroyed automatically when the component is disconnected?

Can't add this as a suggestion since `disconnectedCallback()`'s lines are not in the PR.

Reply:
I don't think so since we're adding this in the constructor, not in `connectedCallback()`. Do we only want it to run when the component is connected?

Reply:
I meant, where should we remove the corresponding event listener in that case? We don't do it anywhere now.

Reply:
Why do we need to remove it?

Reply:
I thought it's a good practice to remove event listeners when they are no longer needed. If only they were removed automatically somehow. I can see that the listener is added inside the constructor. Does it mean we don't need to remove such listeners? Honestly, I don't know the answer to this question. 🤷‍♂️

Reply:
When is it no longer needed? What is the point when we no longer need to listen to prop changes?

Reply:
I guess we don't need to remove the listener. :)